test/tests/functions/baselinecorrection/twenty_sines.i

```diff
@@ -0,0 +1,43 @@
+# A comprehensive test of the BaselineCorrection function output. The input
```

Comment:
Or maybe just keep twenty sines scaled and remove half sine scaled, half sine, and full sine.